src/util/string.cpp

```diff
@@ -93,7 +93,7 @@ std::wstring utf8_to_wide(const std::string &input)
 	std::wstring out;
 	out.resize(outbuf_size / sizeof(wchar_t));
 
-#if defined(__ANDROID__) || defined(__NetBSD__)
+#if defined(__ANDROID__) || defined(__NetBSD__) || defined(__OpenBSD__)
 	SANITY_CHECK(sizeof(wchar_t) == 4);
```

Comment:
not directly related but can you change this to [`static_assert`](https://en.cppreference.com/w/cpp/language/static_assert) so any mismatch is caught at compile-time?